pyqtorch/matrices.py

```diff
@@ -67,14 +67,38 @@ def PROJMAT(ket: Tensor, bra: Tensor) -> Tensor:
 
 
 def parametric_unitary(
-    theta: torch.Tensor, P: torch.Tensor, I: torch.Tensor, batch_size: int  # noqa: E741
+    theta: torch.Tensor,
+    P: torch.Tensor,
+    identity_mat: torch.Tensor,
```

Comment:
Why do you need this as an argument? Can't you use `IMAT` for `pyqtorch.matrices`?

Reply:
It is for supporting the device and dtype conversions from the Parametric gate.